### ISSUE TITLE
Download not working for censusgeo function with four tuples

### ISSUE THREAD

**Issue by jvac06** [jtleider/censusdata#15]
This library seems to be working really well! Thanks for creating this. 

I did run into an issue when trying to download data with 'censusgeo' with four tuples though. When I run the same 'censusgeo' to get the geographical information for this area using 'censusdata.geographies' this seems to work just fine but not when inserting this into the download method.

I also verified that the API call was working with these parameters using postman, so it seems not to be an issue with the endpoints api.census.gov.

_Working API Get Request: https://api.census.gov/data/2018/acs/acs5?get=NAME,B01001_007E&for=place/remainder+(or+part):*&in=state:12+county:057+county+subdivision:93367_

Here is the line that I tried to run:
`censusdata.download('acs5', 2018, censusdata.censusgeo([('state','12'), ('county', '057'), ('county subdivision', '93367'), ('place/remainder (or part)', '*')]),['B01001_007E'])`

The error that I receive: **KeyError: 'state> county> county subdivision> place/remainder (or part)'**

```
censusdata.download('acs5', 2018, censusdata.censusgeo([('state','12'), ('county', '057'), ('county subdivision', '93367'), ('place/remainder (or part)', '*')]),['B01001_010E'],endpt='new')
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/IPython/core/formatters.py in __call__(self, obj)
    700                 type_pprinters=self.type_printers,
    701                 deferred_pprinters=self.deferred_printers)
--> 702             printer.pretty(obj)
    703             printer.flush()
    704             return stream.getvalue()

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/IPython/lib/pretty.py in pretty(self, obj)
    392                         if cls is not object \
    393                                 and callable(cls.__dict__.get('__repr__')):
--> 394                             return _repr_pprint(obj, self, cycle)
    395 
    396             return _default_pprint(obj, self, cycle)

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/IPython/lib/pretty.py in _repr_pprint(obj, p, cycle)
    698     """A pprint that just redirects to the normal repr function."""
    699     # Find newlines and replace them with p.break_()
--> 700     output = repr(obj)
    701     lines = output.splitlines()
    702     with p.group():

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/core/frame.py in __repr__(self)
    685             line_width=width,
    686             max_colwidth=max_colwidth,
--> 687             show_dimensions=show_dimensions,
    688         )
    689 

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/core/frame.py in to_string(self, buf, columns, col_space, header, index, na_rep, formatters, float_format, sparsify, index_names, justify, max_rows, min_rows, max_cols, show_dimensions, decimal, line_width, max_colwidth, encoding)
    818                 line_width=line_width,
    819             )
--> 820             return formatter.to_string(buf=buf, encoding=encoding)
    821 
    822     # ----------------------------------------------------------------------

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/io/formats/format.py in to_string(self, buf, encoding)
    912         encoding: Optional[str] = None,
    913     ) -> Optional[str]:
--> 914         return self.get_result(buf=buf, encoding=encoding)
    915 
    916     def to_latex(

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/io/formats/format.py in get_result(self, buf, encoding)
    519         """
    520         with self.get_buffer(buf, encoding=encoding) as f:
--> 521             self.write_result(buf=f)
    522             if buf is None:
    523                 return f.getvalue()

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/io/formats/format.py in write_result(self, buf)
    821         else:
    822 
--> 823             strcols = self._to_str_columns()
    824             if self.line_width is None:  # no need to wrap around just print
    825                 # the whole frame

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/io/formats/format.py in _to_str_columns(self)
    717         # may include levels names also
    718 
--> 719         str_index = self._get_formatted_index(frame)
    720 
    721         if not is_list_like(self.header) and not self.header:

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/io/formats/format.py in _get_formatted_index(self, frame)
   1057             )
   1058         else:
-> 1059             fmt_index = [index.format(name=self.show_row_idx_names, formatter=fmt)]
   1060 
   1061         fmt_index = [

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/core/indexes/base.py in format(self, name, formatter, **kwargs)
    964             return header + list(self.map(formatter))
    965 
--> 966         return self._format_with_header(header, **kwargs)
    967 
    968     def _format_with_header(self, header, na_rep="NaN", **kwargs):

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/core/indexes/base.py in _format_with_header(self, header, na_rep, **kwargs)
    978 
    979         if is_object_dtype(values.dtype):
--> 980             result = [pprint_thing(x, escape_chars=("\t", "\r", "\n")) for x in values]
    981 
    982             # could have nans

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/core/indexes/base.py in (.0)
    978 
    979         if is_object_dtype(values.dtype):
--> 980             result = [pprint_thing(x, escape_chars=("\t", "\r", "\n")) for x in values]
    981 
    982             # could have nans

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/io/formats/printing.py in pprint_thing(thing, _nest_lvl, escape_chars, default_escapes, quote_strings, max_seq_items)
    228         result = "'{thing}'".format(thing=as_escaped_string(thing))
    229     else:
--> 230         result = as_escaped_string(thing)
    231 
    232     return result

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/pandas/io/formats/printing.py in as_escaped_string(thing, escape_chars)
    204             escape_chars = escape_chars or tuple()
    205 
--> 206         result = str(thing)
    207         for c in escape_chars:
    208             result = result.replace(c, translate[c])

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/censusdata/censusgeo.py in __str__(self)
    128                         return 'Summary level: ' + self.sumlevel() + ', ' + '> '.join([geo[0]+':'+geo[1] for geo in self.geo])
    129                 else:
--> 130                         return self.name + ': Summary level: ' + self.sumlevel() + ', ' + '> '.join([geo[0]+':'+geo[1] for geo in self.geo])
    131 
    132         def params(self):

~/.local/share/virtualenvs/RCVFlorida-m3qDY6vb/lib/python3.7/site-packages/censusdata/censusgeo.py in sumlevel(self)
    155                 Returns:
    156 			str: String representing the summary level code for this object's geographic level, e.g., '050' for 'state> county'."""
--> 157                 return self.sumleveldict[self.hierarchy()]
    158 
    159         def request(self):

KeyError: 'state> county> county subdivision> place/remainder (or part)'```

**Comment by jtleider** [comment]
Thanks for noting this issue. It looks like the naming of the geography hierarchy has changed slightly since I wrote the censusgeo code. That will have to be updated to fix this.

**Comment by jtleider** [comment]
Fixed in version 1.8. Thanks again for pointing this out!

**Comment by jvac06** [comment]
My pleasure!

I just tried it again, it seems to work great. Thanks again for creating this!
